docs/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.py

```diff
@@ -0,0 +1,158 @@
+"""
+QBO Phase plots and QBO Index: UserScript, Stat-Analysis
+==========================================================================
+
+model_applications/
+s2s_stratosphere/
+UserScript_fcstGFS_obsERA_StratosphereQBO.py
+
+"""
+
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# This use case performs several differenc calculations and creates multiple plots
+# to evaluate QBO.  First, the zonal and meridional means on U from -10 S to 10N 
+# latitude are computed.  Then, an EOF analysis is performed on the zonal and meridional
+# mean data and and two Phase diagrams of QBO are created.  The two phase diagrams are 
+# phase circuits which gives one plot for each start date listed in the configuration file 
+# for model and observations, and another phase plot for the whole time period.  Additionally
+# the zonal and meridional mean at 30 and 50mb are output as time series, and the bias (ME) 
+# is computed.  The 30 and 50mb U time series are also plotted.
+#
+
+##############################################################################
+# Datasets
+# --------
+#
+# GFS 0-24 hour forecasts for 10/2017 - 2/2018
+# ERA: 30 year climatology for EOFs and 10/2017 - 2/2018  
+#
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case runs the UserScript wrapper tool to run a user provided script,
+# in this case, stratosphere_qbo_driver.py to compute the zonal and meridional
+# mean, compute anomalies, create EOFs, create Phase diagrams, output U at 30 and
+# 50mb, and create time series of 30 and 50mb winds.  StatAnalysis is then run on 
+# the 30 and 50mb winds to compute the bias (ME).
+#
+
+##############################################################################
+# METplus Workflow
```

Comment:
Try to move some of the current material in the scientific objective area to this one. What is currently here is good (you should put the, "The following tools are run once:..." part in the METplus Components section), but adding all of the handoffs and computations being performed would strengthen this significantly.